### ISSUE TITLE
Trigger installation issues

### ISSUE THREAD

**Issue by francisjes** [trigger/trigger#308]
i installed trigger tool in my Ubuntu PC. but after that i am not able to see the default files like settings.py netdevices.csv etc under trigger folder even i tried to copy from conf folder. but getting below message
root@PC:/usr# cd /etc/trigger/
root@PC:/etc/trigger# ls
root@PC:/etc/trigger#
root@PC:/etc/trigger# sudo cp conf/trigger_settings.py /etc/trigger/settings.py
cp: cannot stat âconf/trigger_settings.pyâ: No such file or directory
root@HPC:/etc/trigger#

please provide the complete process of installation of Trigger tools

**Comment by tcuthbert** [comment]
Looks like something went wrong with your copy there `cp: cannot stat âconf/trigger_settings.pyâ: No such file or directory`

**Comment by tcuthbert** [comment]
What's the output of?:

```
cat /etc/trigger/settings.py
stat /etc/trigger/settings.py
```

**Comment by francisjes** [comment]
Hi,

Please find the required output

root@PC:~# cd /etc/trigger/

root@PC:/etc/trigger#

root@PC:/etc/trigger# ls

root@PC:/etc/trigger#

root@PC:~# cat /etc/trigger/settings.py

cat: /etc/trigger/settings.py: No such file or directory

root@PC:~#

root@PC:~#

root@PC:~# stat /etc/trigger/settings.py

stat: cannot stat â/etc/trigger/settings.pyâ: No such file or directory

root@PC:~#

**Comment by tcuthbert** [comment]
What happens when you run

```
cd /etc/trigger
sudo touch afile
ls
```

**Comment by francisjes** [comment]
Yes am able to create new files. Basically I need default settings files in
trigger folder

root@HMC-NET-PC:/etc/trigger# cd /etc/trigger

root@HMC-NET-PC:/etc/trigger# sudotouch afile

root@HMC-NET-PC:/etc/trigger# ls

afile

root@HMC-NET-PC:/etc/trigger#

**Comment by tcuthbert** [comment]
OK. As there is an issue with `cp` we could try two things:

1. from `/etc/trigger` do the following `touch settings.py` then cut and paste the contents of `trigger_settings.py` into `settings.py` and save.

2. set your TRIGGER_SETTINGS environment variable to an alternate location. You could test this like so:
```
export TRIGGER_SETTINGS=conf/trigger_settings.py
```

I just noticed something. Did you clone/copy the trigger repo into `/etc/trigger/` ? If so dont' do that, clone it to something like `~/.local/src/trigger, /srv/trigger, /opt/trigger or /usr/local/src/trigger` and set yourself as the owner with `chown`.

**Comment by jathanism** [comment]
Closing — Trigger v2.0.0 overhauled the build system to use `pyproject.toml` and entry points, resolving these legacy `setup.py` installation issues.
